tests/click_tests/test_passphrase_mercury.py

```diff
@@ -329,3 +337,16 @@ def test_cycle_through_last_character(
         passphrase = DA_49 + "i"  # for i we need to cycle through "ghi" three times
         input_passphrase(debug, passphrase)
         enter_passphrase(debug)
+
+
+@pytest.mark.setup_client(passphrase=True)
+def test_last_char_timeout(device_handler: "BackgroundDeviceHandler"):
+    with prepare_passphrase_dialogue(device_handler) as debug:
+        for character in CommonPass.MULTI_CATEGORY:
+            # insert a digit
+            input_passphrase(debug, character)
+            # wait until the last digit is hidden
+            time.sleep(DELAY_S)
+            # show the entire PIN
+            show_passphrase(debug)
```

Comment:
what's the point of showing the passphrase after every entry?

Reply:
As a passphrase I used characters, from different character sets, from single/multi character buttons. Showing the passphrase after each entered character helped to cover all possible scenarios, and actually helped me to find a small bug in the implementation.

Reply:
But at this point it's probably overkill, so I've changed it:
https://github.com/trezor/trezor-firmware/pull/4385/commits/7c3615e60b435da4a9fd397b946c32139788948a